tutorial_part1.md

```diff
@@ -1,34 +1,17 @@
-# Title
-...
-This tutorial will be heavily inspired on the ArcBees developer's website [tutorial](TODO).
+# GWTP Beginner's tutorial
+When I first heard about GWT or "gwit", I thought it sounded like some kind of STD, but when I realized it was Java compiling into JavaScript, it got all my attention. Like many people, except the purists (which I respect), I'm not a big fan of JavaScript. Especially when it comes down to writing logic. So here we have GWTP that does the hard work for you and offers you tools that implements all the best practices on how to build web applications with GWT.
```

Comment:
You could lie and say you thought it was "yet another JS framework" but then you realized it was a compile-to-js framework, which got all your attention, because you're not a big fan of JS. :P